README.md

```diff
@@ -1,18 +1,26 @@
 # RTI BioCloud Docker Library
 
-Welcome to the RTI BioCloud Docker Library! This repository serves as a central location for Docker images used by RTI's cloud-based bioinformatics toolkit.
+Welcome to the RTI BioCloud Docker Library! This repository serves as a central location for building and cataloging the Docker images used by RTI's cloud-based bioinformatics toolkit.
 
 ## Repository Structure
 
 The Dockerfiles are organized in the following structure: `<tool-name>/<tool-version>/Dockerfile`. Each tool has its own directory containing the Dockerfile and related files.
-If a software tool does not have a version, with a custom script for example, use `v1` for the tool-version folder.
-
+If a software tool does not have a version, with a custom script for example, use `v1` for the tool-version folder (e.g., RTI_tool/v1/Dockerfile).
 
 <br><br>
 
 # Developers
-## Dockerfile Best Practices Checklist
+Commiting a Dockerfile to this repository will automatically result in the building of a new docker image that is uploaded to Docker Hub. To submit a commit developers should complete the following steps: 
+1. Fork this repository to their account (if not previously done).
+2. Create a branch for the specific docker you working on.
+>A. Create the necessary directory structure when applicable as described above. **Not sure how to make this just an indented bullet**
```

Comment:
you can just type 
* an asterisk followed by a space

Reply:
The spacing is a bit off but this works good enough.